### ISSUE TITLE
man pages seem outdated

### ISSUE THREAD

**Issue by maximedenes** [coq/coq#6920]

**Comment by Zimmi48** [comment]
It would be good if we could have some way of automatically synchronizing the man pages with the rest of the documentation, in particular what's shown with `-help` and what's in the reference manual.

**Comment by chdoc** [comment]
I agree regarding the synchronization between man pages and the manual pages like [this](https://coq.inria.fr/distrib/current/refman/practical-tools/coq-commands.html) one. On the other hand, I suppose that the output of `-help` should be more terse. 

And it doesn't solve the issue, that the manual is outdated as well. For instance `-quick` and `-schedule-vio2vo` are only (very tersely) described in `-help`, but neither in the man page nor the manual. (They are described in a [separate chapter](https://coq.inria.fr/distrib/current/refman/addendum/parallel-proof-processing.html), but not as part of the tools chapter of the manual linked above.)

**Comment by Humm42** [comment]
I believe there should only be one reference documentation for the programs. I think that should be the man pages and a reference book should only cover the basic invocation and otherwise defer to the man pages, but if the reference book must cover the programs’ usage as well, then there should only be a single man page for all the Coq programs containing nothing but a pointer to the reference book.

**Comment by Humm42** [comment]
Automatic synchronization between the man pages and `-help` output doesn’t seem great, because ideally, the man pages would say more about everything than `-help` output. Even if a `-help` output needs a description of every option, it shouldn’t be more than a few words. And the man page certainly should describe what each option does in all the detail.

**Comment by Alizter** [comment]
One suggestion is for `--help` to display the man page just like `cmdliner` does. `coqc` isn't meant to be used manually anyway, so if you have to look up options, you are most likely interested in the manual.

**Comment by Zimmi48** [comment]
Since we haven't been able to progress on this issue in so long, I wouldn't mind something close to what is proposed above, i.e., `--help` printing a terse man page which just lists the existing options for the command and links to the relevant page of the documentation for more information...
